include/deal.II/matrix_free/dof_info.h

```diff
@@ -195,6 +195,7 @@ namespace internal
       void
       read_dof_indices(
         const std::vector<types::global_dof_index> &local_indices,
+        const std::vector<types::global_dof_index> &local_indices_plain,
```

Comment:
Can we pick more suggestive names? I would call the second argument `local_indices`, the first one `local_indices_resolved`, and then expand the documentation to explain that `resolved` means that hanging nodes have been tracked back to the indices on the coarse side. (We could think about `local_indices_hanging_nodes_resolved` but that name is too long in my opinion, at least if we insist on 80 character limits.)